product_docs/docs/efm/5/04_configuring_efm/01_cluster_properties.mdx

```diff
@@ -121,7 +122,7 @@ Use the properties in the `efm.properties` file to specify connection, administr
 | [pgpool.bin](#pgpool_enable)                                  |       |       |                                       | Required if pgpool.enable is set to true. **Available in Failover Manager 4.1 and later. **                                                                                                                     |
 | [script.load.balancer.attach](#script_load_balancer)          |       |       |                                       | Example: script.load.balancer.attach= /&lt;path>/&lt;attach_script> %h %t                                                                                      |
 | [script.load.balancer.detach](#script_load_balancer)          |       |       |                                       | Example: script.load.balancer.detach= /&lt;path>/&lt;detach_script> %h %t                                                                                      |
-| [detach.on.agent.failure](#detach_on_agent_failure)           |       |       | true                                  | Set to false if you want to keep a running primary database attached.  **Available in Failover Manager 4.2 and later.**                                                                                   |
+| [detach.on.agent.failure](#detach_on_agent_failure)           |       |       | false                                 |                                                                                                                                                                |
```

Comment:
Because we now have a sane value for this property, I don't think we need any explanatory info here. It's a property that no one should use except in very specific circumstances. If even then....

Reply:
I still think we should say something here as the empty field might generate more questions. 

We could either say this property is not to be used in production. 

Or say this is an outdated value replaced by [sane value here].

Or we should remove the line completely if we really don't want users using it.

Reply:
I've added some text here. Hopefully it makes sense.